### DISCUSSION TITLE
Modding configs for 1.6

### DISCUSSION THREAD

**Discussion by ghost** [redeclipse/base#1389] [Help and Support]
Hi, I am a Mac user playing Red Eclipse 1.6 (it is too weak to run 2.0). I figured out how to load custom maps, but is there a way to store custom variable configurations (e.g battle royale or sniper one shot), etc? I heard there was, but where is the directory on Mac and how to make a custom cfg file?

**Comment by Calinou** [comment]
After customizing game variables, using `/writevars <output name without extension>` will write a `.cfg` file containing all *game* variables (this excludes your client's settings). You can then load it using `/exec <output name>.cfg`.
